src/cli_config.rs

```diff
@@ -143,30 +156,29 @@ pub fn cli_parse() -> CliConfig {
             .long("keys-file")
             .help("Path to the file where DHT keys are stored")
             .takes_value(true)
-            .required_unless("secret-key")
-            .conflicts_with("secret-key"))
+            .conflicts_with("secret-key")
+            .required_unless_one(&["config", "secret-key"]))
         .arg(Arg::with_name("bootstrap-node")
             .short("b")
             .long("bootstrap-node")
             .help("Node to perform initial bootstrap")
             .multiple(true)
             .takes_value(true)
             .number_of_values(2)
-            .value_names(&["public key", "address"]))
+            .value_names(&["public key", "address"])
+            .required_unless("config"))
```

Comment:
Why is it required?

Reply:
Ah. If we use lan_discovery, it is not required.